src/shapes/text.class.ts

```diff
@@ -1141,16 +1131,7 @@ export class Text<
     if (shortCut) {
       // render all the line in one pass without checking
       // drawingLeft = isLtr ? left : left - this.getLineWidth(lineIndex);
-      this._renderChar(
-        method,
-        ctx,
-        lineIndex,
-        0,
-        line.join(''),
-        left,
-        top,
-        lineHeight
-      );
+      this._renderChar(method, ctx, lineIndex, 0, line.join(''), left, top);
```

Comment:
there is some weird reason why lineHeight was there, i ll check it and report back

Reply:
I wasn't sure, saw it wasn't in the signature any longer

Reply:
We can change the signature to a destructed param, then adding/removing a key is not biggy in terms of subclassing/overriding